### DISCUSSION TITLE
[Bug]Subtitles not showing in Safari

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#635] [Q&A]
Specifically in Safari, subtitles will not show until the subtitles are turned off and then on again.

**Comment by mgogoulos** [comment]
Sounds like a frontend bug, we'll reproduce in Safari and check if there's any quick fix. Did you try this on other browsers too? 
Thanks for reporting

**Comment by Dan1ell** [comment]
Works fine in Chrome, Opera, Firefox, and Brave.
